ipad/ViewControllers/Shift/ShiftViewController.swift

```diff
@@ -177,12 +193,14 @@ class ShiftViewController: BaseViewController {
     @objc func completeAction(sender: UIBarButtonItem) {
         guard let model = self.model else { return }
         self.dismissKeyboard()
+
+        self.updateStatuses()
         // if can submit
         var alertMessage = "This shift and the inspections will be uploaded when possible"
         if model.shiftStartDate < Calendar.current.startOfDay(for: Date()) {
             alertMessage += "\n\n You've entered a date that occurred before today. If this was intentional, no problem! Otherwise, please double-check the entered date: \n\(model.shiftStartDate.stringShort())"
         }
-        if canSubmit() {
+        if canSubmit() && model.inspections.allSatisfy({ $0.formDidValidate }) {
```

Comment:
This addition of the if statement prevents the user to submit an error containing inspection (avoiding our open loop validation problem)